packages/lism-ui/src/components/Details/_style.css

```
@@ -30,15 +30,15 @@
     display: inline-grid;
     background-color: currentColor;
     mask: var(--svg) no-repeat center center / contain;
-    scale: var(--_icon-scale);
+    scale: var(--icon-scale);
 
     transition: transform var(--duration);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)